### PR TITLE
TRT-668: linter updates

### PR DESCRIPTION
**Pull request by neisw**

### CODE REVIEW

File: pkg/prowloader/prow.go [hunk 1]
```diff
@@ -181,6 +181,8 @@ func fetchJobsJSON(prowURL string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+
```
Comment (neisw) [inline pkg/prowloader/prow.go:184]: Slipped this in, I think we need it but if not let me know

File: pkg/prowloader/prow.go [hunk 2]
```diff
@@ -181,6 +181,8 @@ func fetchJobsJSON(prowURL string) ([]byte, error) {
...
+	defer resp.Body.Close()
```
Comment (neisw) [inline pkg/prowloader/prow.go:185]: Slipped this in, I think we need it but if not let me know

**Comment by stbenjam** [comment]
I think I'd prefer to understand why it's failing, disabling type checks and renaming imports points to something having changed that needs understanding before we merge this.

**Pull request closed without merging**